about.html

```diff
@@ -0,0 +1,304 @@
+                <div class="col-8">
+                    <ul>
+                        <li>
+                            <a href="index.html">Home</a>
+                        </li>
+                        <li>
+                            <a href="##">Program</a>
+                        </li>
+                        <li>
+                            <a href="##">Join</a>
+                        </li>
+                        <li>
+                            <a href="##">Sponsor</a>
+                        </li>
+                        <li>
+                            <a href="##">News</a>
+                        </li>
+                        <li>
+                            <a href="##">Follow us</a>
+                        </li>
+                    </ul>
+                </div>
+    
+            </div>
+        </nav>
+       
+        <!--main section-->
+        <div class="container-fluid main-section">
+             
+
+            <div class="container hambuger" id="hambuger_about">
+                <div class="row">
+                    <div class="col-12 hambuger-menu">
+                        <img src="./images/mobile_menu.png" alt="menu-bar">
+
+                    </div>
+                </div>
+
+                <!--Hamburger drop down-->
+                <div class="container-fluid hide-menu" id="hide_menu_about">
+                    <div class="row">
+                        <div class="col-12 drop-down-menu">
+                            
+                            <ul>
+                                <li>
+                                    <a href="index.html">Home</a>
+                                </li>
+                                <li>
+                                    <a href="##">Program</a>
+                                </li>
+                                <li>
+                                    <a href="##">Join</a>
+                                </li>
+                                <li>
+                                    <a href="##">Sponsor</a>
+                                </li>
+                                <li>
+                                    <a href="##">News</a>
+                                </li>
+                                <li>
+                                    <a href="##">Follow us</a>
+                                </li>
+                            </ul>
+                            <span class="closeButton" id="close_button_about">&times;</span>
+                        </div>
+                    </div>
+                </div>
+
+
+            </div>
+            <div class="row">
+
+                <div class="container">
+                    <div class="row">
+                        <div class="col-3 desk-div"></div>
+                        <div class="col-9">
+                            <h1 class="about_header">RR Conference.
+                                <br> <span class="about_titlespan">South Africa conference 2023</span>
```

Comment:
- [ ] This text should be centered in the container following the original design. kindly consider fixing it.
your design
![h](https://user-images.githubusercontent.com/96683843/226442195-c4a8b963-d5ad-494c-9615-a8f8a7ff021a.jpg)